source/_docs/metrics.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Metrics in the Site Dashboard
+description: Measure your site's traffic with the Metrics tool, found in the Live environment of the Site Dashboard.
+tags: [performance, logs]
+---
+## Access Metrics
+1. Navigate to **<span class="glyphicons glyphicons-charts"></span> Metrics** within the **<span class="glyphicons glyphicons-cardio"></span> Live** environment of your Site Dashboard.
+2. Toggle available date ranges by clicking **Day**, **Week**, or **Month**:
+  ![Charts for pages served and visits within the Metrics tool of the Site Dashboard](/source/docs/assets/images/dashboard/metrics-graphs.png)
+
+## Available Metrics
+{% include("content/traffic-dl.html")%}
+
+For example, imagine two visitors go to `example.com`. The first visitor exits from the home page, but the second visitor continues to browse two additional pages. This would be tracked as 2 visits and 4 pages served.  
```

Comment:
This is great. Do you think we should expand on this example to make it clear that if someone visits again tomorrow it will count as another visit? Something like, "Note that visits and visitors aren't the same thing. If one of the visitors returns again the next day, it would count as a third  visit, but there'd be only two visitors."